adapters/MythicMobsAdapter/src/main/kotlin/com/caleb/typewriter/mythicmobs/entries/event/MythicMobDeathEventEntry.kt

```diff
@@ -0,0 +1,33 @@
+package com.caleb.typewriter.mythicmobs.entries.event
+
+import io.lumine.mythic.bukkit.events.MythicMobDeathEvent
+import lirand.api.extensions.server.server
+import me.gabber235.typewriter.adapters.Colors
+import me.gabber235.typewriter.adapters.Entry
+import me.gabber235.typewriter.adapters.modifiers.Help
+import me.gabber235.typewriter.entry.*
+import me.gabber235.typewriter.entry.entries.EventEntry
+import me.gabber235.typewriter.utils.Icons
+import java.util.*
+
+
+@Entry("on_mob_die", "When a player kill a MythicMobs mob.", Colors.YELLOW, Icons.SQUARE_CHECK)
```

Comment:
Square Check feels like a weird Icon for this. Have a look at [font awesome icons](https://fontawesome.com/search?o=r&m=free) to see the available icons (only the free ones).